joomla/joomla-cms
# Fixed ReCaptcha position in user registration form

Pull Request for Issue #37698

### Testing Instructions
Enable ReCaptcha and custom fields in registration form and check the position of ReCaptcha. It should be placed at the bottom of the form, right before Register button.


### Actual result BEFORE applying this Pull Request
The ReCaptcha is placed before custom fields, in the middle of the form


### Expected result AFTER applying this Pull Request
The ReCaptcha should be placed before the Register button
